spec/views/teams/show.html.erb_spec.rb

```diff
@@ -11,4 +11,34 @@
     expect(rendered).to have_content(@team.kind_of_sport, count: 1)
     expect(rendered).to have_content(@team.description, count: 1)
   end
+
+  it " shows assign ownership button for mere members" do
+    user = FactoryBot.create :user    
```

Comment:
See the comment on the `team_member_spec.rb` file: you can factor out the redundant creation of the user via a `describe` and `let` construct.